### PR TITLE
OpenEphys updates for NEO and PI

### PR DESCRIPTION
**Pull request by alejoe91**
Update kwargs and probe handling. 

Based on:
- probeinterface: https://github.com/SpikeInterface/probeinterface/pull/121
- NEO: https://github.com/NeuralEnsemble/python-neo/pull/1159

### CODE REVIEW

**Comment by samuelgarcia** [comment]
Waiting for :
https://github.com/NeuralEnsemble/python-neo/pull/1159
https://github.com/NeuralEnsemble/python-neo/pull/1164

And then we can merge this.

**Comment by alejoe91** [comment]
This is failing because of the cached `ephy_testing_folder`. Tests pass locally

File: spikeinterface/extractors/neoextractors/openephys.py [hunk 1]
```diff
@@ -96,32 +103,59 @@ class OpenEphysBinaryRecordingExtractor(NeoBaseRecordingExtractor):
     name = "openephys"
     has_default_locations = True
 
-    def __init__(self, folder_path, stream_id=None, stream_name=None, block_index=None, all_annotations=False):
-        neo_kwargs = self.map_to_neo_kwargs(folder_path)
-        NeoBaseRecordingExtractor.__init__(self, stream_id=stream_id, 
+    def __init__(self, folder_path, load_sync_channel=False, experiment_names=None,
+                 stream_id=None, stream_name=None, block_index=None, all_annotations=False):
+        neo_kwargs = self.map_to_neo_kwargs(folder_path, load_sync_channel, experiment_names)
```
Comment (samuelgarcia) [inline spikeinterface/extractors/neoextractors/openephys.py:108]: neo_kwargs = self.map_to_neo_kwargs(folder_path, load_sync_channel=load_sync_channel, experiment_names=experiment_names)

Comment (alejoe91) [reply spikeinterface/extractors/neoextractors/openephys.py:108]: I think that's ok

**Comment by h-mayorquin** [comment]
This should close #923.

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**